### PR TITLE
Add tfm-ror51 repo to foreman-release-scl

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: foreman-release-scl/foreman-release-scl.spec [hunk 1]
```diff
@@ -19,7 +20,18 @@ This meta-package depends on those packages that contain the Yum repository
 configuration files required for Foreman.  It's designed for use on Red Hat
 Enterprise Linux rebuilds, such as CentOS.
 
+%install
+rm -rf %{buildroot}
+
+install -d -m 0755 %{buildroot}%{_sysconfdir}/yum.repos.d
+
+install -m 644 %{SOURCE0} %{buildroot}%{_sysconfdir}/yum.repos.d/
```
Comment (ekohl) [inline foreman-release-scl/foreman-release-scl.spec]: You can use ```-D``` to automatically create the directory

File: foreman-release-scl/foreman-release-scl.spec [hunk 2]
```diff
@@ -19,7 +20,18 @@ This meta-package depends on those packages that contain the Yum repository
 configuration files required for Foreman.  It's designed for use on Red Hat
 Enterprise Linux rebuilds, such as CentOS.
 
+%install
+rm -rf %{buildroot}
```
Comment (ekohl) [inline foreman-release-scl/foreman-release-scl.spec]: Shouldn't this only be in the clean step?

File: foreman-release-scl/foreman-release-scl.spec [hunk 3]
```diff
@@ -19,7 +20,18 @@ This meta-package depends on those packages that contain the Yum repository
...
+
+install -d -m 0755 %{buildroot}%{_sysconfdir}/yum.repos.d
+
+install -m 644 %{SOURCE0} %{buildroot}%{_sysconfdir}/yum.repos.d/
+
+%clean
```
Comment (ekohl) [inline foreman-release-scl/foreman-release-scl.spec]: I wonder if this isn't the default implementation and thus redundant.

File: foreman-release-scl/tfm-ror51.repo [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# As defined by https://copr.fedorainfracloud.org/coprs/g/theforeman/tfm-ror51/repo/epel-7/group_theforeman-tfm-ror51-epel-7.repo
+[group_theforeman-tfm-ror51]
+name=Copr repo for tfm-ror51 owned by @theforeman
+baseurl=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/epel-7-$basearch/
+type=rpm-md
+skip_if_unavailable=True
+gpgcheck=1
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/pubkey.gpg
```
Comment (ekohl) [inline foreman-release-scl/tfm-ror51.repo]: Would it be a good idea to ship the GPG key in the package?

File: foreman-release-scl/tfm-ror51.repo [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# As defined by https://copr.fedorainfracloud.org/coprs/g/theforeman/tfm-ror51/repo/epel-7/group_theforeman-tfm-ror51-epel-7.repo
+[group_theforeman-tfm-ror51]
+name=Copr repo for tfm-ror51 owned by @theforeman
+baseurl=https://copr-be.cloud.fedoraproject.org/results/@theforeman/tfm-ror51/epel-7-$basearch/
+type=rpm-md
+skip_if_unavailable=True
```
Comment (ekohl) [inline foreman-release-scl/tfm-ror51.repo]: Won't this possibly cause silent errors?

File: foreman-release-scl/tfm-ror51.repo (hunk 4 above)
Comment (ehelms) [reply foreman-release-scl/tfm-ror51.repo]: I suppose we can? Though, if the GPG key is unavailable, so too will the packages.

**Comment by ehelms** [comment]
Updated based on comments

**Pull request merged**